### PR TITLE
Implemented async methods

### PR DESCRIPTION
**Pull request by amiryonatan**
Same code as before, just had problems to merge the code for some reason

### CODE REVIEW

File: StatsdClient/IStatsd.cs [hunk 1]
```diff
@@ -9,45 +10,47 @@ public interface IStatsd
...
         /// <summary>
         /// Log a gauge value
         /// </summary>
-        void LogGauge(string name, int value);
+        Task LogGaugeAsync(string name, long value);
+
         /// <summary>
         /// Log a latency / Timing
         /// </summary>
-        void LogTiming(string name, int milliseconds);
```
Comment (niemyjski) [inline StatsdClient/IStatsd.cs]: I don't think this should break anyone.

Comment (amiryonatan) [inline StatsdClient/IStatsd.cs]: I agree

**Comment by niemyjski** [comment]
Thank you for your pull request!

**Comment by amiryonatan** [comment]
No problem .. When do you think you will release a new nuget version ?

**Comment by niemyjski** [comment]
I don't have access to nuget / build process (we should set up a free one on app veyor). @lukevenediger can you take a look into this. I haven't done any testing on this commit but it looks good.

**Pull request merged**